docs/reference/commands/5ttgen.rst

```diff
@@ -213,9 +213,9 @@ Options specific to the 'fsl' algorithm
 
 - **-premasked** Indicate that brain masking has already been applied to the input image
 
-- **-first_dir /path/to/first/dir** use output of FSL FIRST if it has been previously run on input T1-weighted image, in the SAME SPACE as input T1
+- **-first_dir /path/to/first/dir** use output of FSL FIRST if it has been previously run on input T1-weighted image, in the SAME SPACE as input T1. Filename prefix should be first-
```

Comment:
The main reason why one would be using a capability such as this is if it is intended for these data to be used in multiple subsequent pipelines; demanding a specific naming convention may undermine such. What would be preferable would be:

1. Query the contents of the filesystem directory;
2. Search for prefix strings for which all of the requisite file names are present;
3. Proceed if the number of such unique prefixes is 1.